bert/yamls/main/mcloud_run.yaml

```diff
@@ -45,7 +45,7 @@ parameters:
   train_loader:
     name: c4
     dataset:
-      remote: *data_remote
+      remote:                    # Supply a path to your remote dataset
```

Comment:
Seems like the `*data_remote` got nuked here, which is probably not an intended consequence of replacing `null` with empty.

Reply:
Good catch. Looks like a yamlfmt bug. Fixed via workaround.

Reply:
Is it possible to have `parameters.data_remote: &data_remote` (so that the value is blank but the linker `&data_remote` is still present), so that we can keep `parameters.train_loader.dataset.remote: *data_remote` and `parameters.eval_loader.dataset.remote: *data_remote`?

If not, the `parameters.data_remote` makes it more tedious to fill in the YAML (whereas it was included to do the opposite).